nengo_loihi/conftest.py

```diff
@@ -111,11 +111,18 @@ def _allclose(a, b, rtol=1e-5, atol=1e-8, equal_nan=False):
 
 @pytest.fixture
 def assert_allclose(request):
-    def _assert_allclose(a, b, rtol=1e-5, atol=1e-8, equal_nan=False):
+    def _assert_allclose(a, b, rtol=1e-5, atol=1e-8, xtol=0, equal_nan=False):
```

Comment:
Once this fixture is using `allclose`, it'd be good to have the `xtol` implementation in `allclose` instead (since that seems like something that would be useful in lots of places).

Reply:
Also a test/documentation for the `xtol` behaviour would be great, since it's not obvious and the implementation is a bit complicated.